source/languageHandler.py

```diff
@@ -147,6 +148,13 @@ def getWindowsLanguage():
 	return localeName
 
 def setLanguage(lang):

```

Comment:
Could you add typing for `lang`? I assume it is `str`. An example value is also handy, eg "en-us"